src/features/documents/DocumentList.tsx

```diff
@@ -210,7 +210,7 @@ export function DocumentList({ isDetailOpen = false }: { isDetailOpen?: boolean
                     <span className="td id">{key}</span>
                     {!isDetailOpen && (
                       <span className="td updated">
-                        {moment.unix(updatedAt).format(`${use24HourClock ? 'MMM D, H:mm' : 'MMM D, h:mm A'}`)}
+                        {format(fromUnixTime(updatedAt), `${use24HourClock ? 'MMM d, h:mm' : 'MMM d, h:mm A'}`)}
```

Comment:
```suggestion
                        {format(fromUnixTime(updatedAt), `${use24HourClock ? 'MMM d, h:mm' : 'MMM d, h:mm aa'}`)}
```

`A` should be changed to `aa` according to the docs.

https://date-fns.org/v3.6.0/docs/format
<img width="716" alt="Screenshot 2024-04-15 at 8 47 52 PM" src="https://github.com/yorkie-team/dashboard/assets/2059311/eece70cf-f00d-4a96-8f22-e42e0ec5c44e">

Reply:
Thanks for your review! I changed it